.github/workflows/test.yml

```diff
@@ -82,7 +79,7 @@ jobs:
               repo: "git+https://github.com/vllm-project/vllm --tag v0.10.2"
             test_suite:
               name: "backward compat"
-              markers: "compat or (cpu and basic)"
+              markers: "compat or (cpu and basic) and not quantized and not static_batching"
```

Comment:
Is this relying on operator precedence?
I presume we want all `compat` tests to run so we may want to be explicit:
```
compat or (cpu and basic and not quantized and not static_batching)"
```